django/core/serializers/python.py

```diff
@@ -184,11 +185,17 @@ def m2m_convert(v):
         yield base.DeserializedObject(obj, m2m_data)
 
 
-def _get_model(model_identifier):
+def _get_model(model_identifier, project_apps):
     """
     Helper to look up a model from an "app_label.model_name" string.
     """
     try:
-        return apps.get_model(model_identifier)
+        if project_apps:
+            model_import_split = model_identifier.split(".")
+            app_identified = model_import_split[0]
+            model_identifier = model_import_split[1]
+            return project_apps.get_model(app_identified,  model_identifier)
```

Comment:
Does this work with the standard get_model method? Then we can just select the apps module to use and have the same code.